swal-forms.js

```diff
@@ -46,10 +46,13 @@
     addWayToGetFormValuesInDoneFunction: function(swalArgs) {
       var swalFormInstance = this
       var doneFunction = swalArgs[1]
-      swalArgs[1] = function() {
+      swalArgs[1] = function(isConfirm) {
```

Comment:
I believe there might be a misunderstanding with the sweet-alertexample you link to. The way how I understand it works is that the same callback function is called regardless of which button ("ok" or "cancel") is clicked, and this `isConfirm` parameter adds the possibility to handle each scenario. In the example you link, the parameter is used for this purpose, while in the code where you are introducing this `isConfirm` parameter it is not used at all.